docs/api/products.rst

```diff
@@ -452,322 +249,551 @@ Example
+								"path": "b8/d0/c80dabb28dfc53795be8fa88444c.jpeg"
+							}
+						],
+						"_links": {
+							"self": {
+								"href": "/api/v1/products/1"
+							}
+						}
+					},
+					{
+						"name": "Mug \"et\"",
+						"id": 2,
+						"code": "e5e45464-c35f-3c05-b3ea-4743ccafb28e",
+						"attributes": [
+							{
+								"code": "mug_material",
+								"name": "Mug material",
+								"value": "Invisible porcelain",
+								"type": "text",
+								"id": 2
+							}
+						],
+						"variants": [
+							{
+								"id": 4,
+								"on_hold": 0,
+								"tracked": false
+							},
+							{
+								"id": 5,
+								"on_hold": 0,
+								"tracked": false
+							},
+							{
+								"id": 6,
+								"on_hold": 0,
+								"tracked": false
+							}
+						],
+						"options": [
+							{
+								"code": "mug_type"
+							}
+						],
+						"associations": [],
+						"translations": {
+							"en_US": {
+								"locale": "en_US",
+								"id": 2,
+								"name": "Mug \"et\"",
+								"slug": "mug-et",
+								"description": "Omnis perspiciatis quia aperiam magni occaecati",
+								"short_description": "Laboriosam blanditiis."
+							}
+						},
+						"product_taxons": [
+							{
+								"id": 2,
+								"taxon": {
+									"name": "Mugs",
+									"id": 2,
+									"code": "mugs",
+									"children": []
+								},
+								"position": 1
+							}
+						],
+						"main_taxon": {
+							"name": "Mugs",
+							"id": 2,
+							"code": "mugs",
+							"children": []
+						},
+						"reviews": [],
+						"average_rating": 0,
+						"images": [
+							{
+								"id": 3,
+								"code": "main",
+								"path": "bc/93/e2986698753c469277570a416ad2.jpeg"
+							},
+							{
+								"id": 4,
+								"code": "thumbnail",
+								"path": "86/78/092031fdb34daeac17f7da621424.jpeg"
+							}
+						],
+						"_links": {
+							"self": {
+								"href": "/api/v1/products/2"
+							}
+						}
+					}
+				],
+				"created_at": "2017-02-01T14:39:29+0100",
+				"updated_at": "2017-02-01T14:39:29+0100"
+			}
+		],
+		"translations": {
+			"en_US": {
+				"locale": "en_US",
+				"id": 62,
+				"name": "Theme Mug",
+				"slug": "theme-mug"
+			},
+			"pl_PL": {
+				"locale": "pl_PL",
+				"id": 63,
+				"name": "Kubek z motywem",
+				"slug": "kubek-z-motywem"
+			}
+		},
+		"product_taxons": [
+			{
+				"id": 76,
+				"taxon": {
+					"name": "Mugs",
+					"id": 2,
+					"code": "mugs",
+					"children": []
+				},
+				"position": 15
+			}
+		],
+		"main_taxon": {
+			"name": "Mugs",
+			"id": 2,
+			"code": "mugs",
+			"children": []
+		},
+		"reviews": [],
+		"average_rating": 0,
+		"images": [
+			{
+				"id": 1,
+				"type": "ford",
+				"path": "b9/65/01cec3d87aa2b819e195331843f6.jpeg"
+			}
+		],
+		"_links": {
+			"self": {
+				"href": "/api/v1/products/62"
+			}
+		}
+	}
+
+.. note::
+
+	The images should be passed in array as an attribute (files) of request. See how it is done in Sylius
```

Comment:
`The images (files) should be passed in an array as an attribute of request. (...)`